### DISCUSSION TITLE
question about charging

### DISCUSSION THREAD

**Discussion by legosteam** [pybricks/support#1580] [Q&A]
while charging our inventor hub, the side light starts out red and then goes to very slowly flashing green after it has been charging awhile. The docs talk about a solid red and solid green light but make no mention of flashing that I could find. What does flashing green mean? I had it print voltage and current while in this state and got a little over 8100 and 100 respectively. How do I interpret these numbers? Thanks!

**Comment by laurensvalk** [comment]
Thank you for asking, we should add that to the docs.

Blinking green means that charging is fully complete. At this point we make it stop charging, in order to protect the battery.

You could unplug it if you like; since it's no longer charging this has the same effect.
